integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap-slim.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: falco-daemonset
+  labels:
+    app: falco-example
+    role: security
+spec:
+  selector:
+    matchLabels:
+      app: falco-example
+      role: security
+  template:
+    metadata:
+      labels:
+        app: falco-example
+        role: security
+    spec:
+      serviceAccount: falco-account
+      initContainers:
+        - name: probeloader
+          image: falcosecurity/probeloader:latest
+          securityContext:
+            privileged: true
+          #env:
+          #  - name: FALCOCTL_FALCO_VERSION
+          #    value: 0.21.0
+          #  - name: FALCOCTL_FALCO_PROBE_URL
+          #    value:
+          #  - name: FALCOCTL_FALCO_PROBE_REPO
+          #    value: "https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/"
```

Comment:
This URL can be left as is for now (but it'll need to be updated soon :))

Reply:
I'm not sure those are working, I propose to fix this later in another PR and unlock this now since we have everything else solved (once suggestions have been applied).
@leodido @maxgio92 WDYT?

Reply:
I agree @leogr